cfme/common/vm.py

```diff
@@ -792,7 +793,15 @@ def create_on_provider(self, timeout=900, find_in_cfme=False, delete_on_failure=
             find_in_cfme: Verifies that VM exists in CFME UI
             delete_on_failure: Attempts to remove VM on UI navigation failure
         """
-        vm = deploy_template(self.provider.key, self.name, self.template_name, **kwargs)
+        try:
+            from polarion_docstrings.parser import DocstringParser
+            assignee = DocstringParser(request.function.__doc__).parse()["assignee"].value
```

Comment:
This metadata can be defined at levels other than the function. There are new methods which merge the meta from all levels, in a given file. I'm using them in #9225